<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(76, 76, 76);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>661</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>239</width>
       <height>659</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>641</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>211</width>
         <height>641</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>811</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_imagen">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_widget">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="myTextEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../../../../Downloads/muestra-cuadrada-y-sÃ­mbolo-del-vector-icono-aislados-en-el-fondo-blanco-133762365.qrc"/>
 </resources>
 <connections/>
</ui>
